Makefile

```diff
@@ -438,15 +415,16 @@ define manifest
 	$(call controller-gen,crd:crdVersions=v1 paths="./..." output:crd:artifacts:config=chart/k8gb/templates/)
 endef
 
-# function retrieves controller-gen path or installs controller-gen@v3.0.0 and retrieve new path in case it is not installed
 define controller-gen
```

Comment:
I think it is also kustomize.

Reply:
@kuritka mind to elaborate? should we cover kustomize installation by `go install`?

Reply:
kustomize's go mod has exclude directives, therefore we can't install it with `go install` as per https://golang.org/issue/43684